andela/mrm-mobile
# #158102870 Integrate CodeClimate and Coveralls

#### What does this PR do?
- Integrates Code Climate and Coveralls For Test reports and code quality checks
#### Description of Task to be completed?
- [x]  Add Repo to Code Climate and configure
- [x]  Add Repo to Coveralls and configure
#### How should this be manually tested?
- clone repo and open in android studio
- Add a feature
- Write Test and push to Github
- Observe coverage and maintainability on top of the README file
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
- story type - Chore
- story title - Integrate code climate
- story id - #158102870
#### Screenshots (if appropriate)
